src/MainMenu.cpp

```diff
@@ -2010,6 +2008,69 @@ u32 MainMenu::OnUpdateOptionsMenu()
 }
 #pragma optimize("", on)
 
+#pragma optimize("s", on)
+void MainMenu::ColorMenuItem(AnmVm *vm, i32 item, i32 subItem, i32 subItemSelected)
+{
+    if (subItem != subItemSelected)
+    {
+        if (!g_Supervisor.cfg.IsSoftwareTexturing())
+        {
+            vm->color.color = COLOR_MENU_ITEM_DEFAULT;
+        }
+        else
+        {
+            g_AnmManager->SetActiveSprite(vm, vm->anotherSpriteNumber);
+        }
+        vm->scaleX = 1.0;
+        vm->scaleY = 1.0;
+        vm->posOffset = D3DXVECTOR3(0.0, 0.0, 0.0);
+    }
+    else
+    {
+        if (!g_Supervisor.cfg.IsSoftwareTexturing())
+        {
+            vm->color.color = COLOR_MENU_ITEM_HIGHLIGHT;
+        }
+        else if (vm->anotherSpriteNumber < 0x122)
+        {
+            g_AnmManager->SetActiveSprite(vm, vm->anotherSpriteNumber + 0x7a);
+        }
+        else
+        {
+            g_AnmManager->SetActiveSprite(vm, vm->anotherSpriteNumber + 0x73);
+        }
+        vm->posOffset = D3DXVECTOR3(-2.0, -2.0, 0.0);
+    }
+
+    if (item != this->cursor)
+    {
+        if ((g_Supervisor.cfg.opts >> GCOS_USE_D3D_HW_TEXTURE_BLENDING & 1) == 0)
+        {
+            vm->color.color = vm->color.color & 0xffffff | 0x80000000;
```

Comment:
I merged the COLOR_ macros, I think this can be written as `COLOR_SET_ALPHA2(128)`